### PR TITLE
conda FAQ

### PR DESCRIPTION
**Pull request by njzjz**

### CODE REVIEW

File: faq/conda.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+:::
+
+An environment is a directory that contains a specific collection of packages that you have installed.
+Different environments are isolated from each other.
+A base environment is required before setuping up multiple environments.
+
+First of all, download and execute [conda-forge installer](https://conda-forge.org/download/):
+
+```sh
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh -O miniforge.sh
```
Comment (caic99) [inline faq/conda.md:22]: Shall we mention Windows users?

File: faq/conda.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+However, sometimes, you may want to install CUDA packages into a machine that doesn't install NVIDIA drivers (for example, the login node of a HPC cluster).
+In this case, you must manually set the environment variable `CONDA_OVERRIDE_CUDA` to tell conda which packages are compatible.
+
+For example, you want to use the installed CUDA packages on another machine where NVIDIA drivers have been installed. Run `nvidia-smi` on that machine, 
+```
++---------------------------------------------------------------------------------------+
+| NVIDIA-SMI 545.29.06              Driver Version: 545.29.06    CUDA Version: 12.3     |
+|-----------------------------------------+----------------------+----------------------+
+```
+the output shows the CUDA version is `12.3`.
```
Comment (caic99) [inline faq/conda.md]: nvidia-smi shows the highest version of CUDA supported by the driver.

```suggestion
the output shows the highest supported CUDA version is `12.3`.
```

File: faq/conda.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+conda create -n my_env python=3.11
+```
+
+Now you can activate the environment using
+
+```sh
+conda activate my_env
+```
+
+If you want to activate the environment in a shell script, using `shell` instead:
```
Comment (caic99) [inline faq/conda.md]: typo?
```suggestion
If you want to activate the environment in a shell script, using `source` instead:
```

File: faq/conda.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+## How to setup a conda/pip base environment?
+
+:::{note}
+The following instructions are for Miniforge but should also work for the offline packages provided by the DeepModeling projects.
+:::
+
+An environment is a directory that contains a specific collection of packages that you have installed.
+Different environments are isolated from each other.
+A base environment is required before setuping up multiple environments.
```
Comment (caic99) [inline faq/conda.md]:
```suggestion
A base environment is required before setting up multiple environments.
```

**Review by caic99 — APPROVED** [review]

File: faq/conda.md (hunk 2 above)
Comment (njzjz) [reply faq/conda.md]: I don't think so... It just shows the CUDA driver version (not the NVIDIA driver). But you can use CUDA toolkit 12.3 when the CUDA driver 12.0 is installed; and you can use CUDA toolkit 11.8 when the CUDA driver 11.2 is installed.

File: faq/conda.md (hunk 1 above)
Comment (njzjz) [reply faq/conda.md:22]: It seems most DeepModeling packages don't support Windows, though...

Comment (njzjz) [reply faq/conda.md:22]: Perhaps we can say, for Windows users, we suggest using WSL

File: faq/conda.md (hunk 2 above)
Comment (njzjz) [reply faq/conda.md]: Perhaps this is another necessary FAQ for CUDA things 🤣

Comment (caic99) [reply faq/conda.md]: > I don't think so... It just shows the CUDA driver version (not the NVIDIA driver). 

I'm not sure if I got you right... [This post](https://stackoverflow.com/questions/53422407/different-cuda-versions-shown-by-nvcc-and-nvidia-smi) said that.

> But you can use CUDA toolkit 12.3 when the CUDA driver 12.0 is installed; and you can use CUDA toolkit 11.8 when the CUDA driver 11.2 is installed.

That's right, and even the CUDA Toolkit is missing (only driver present), conda still installs packages with CUDA support correctly.

Anyway it's all up to you 🤣

File: faq/conda.md (hunk 1 above)
Comment (caic99) [reply faq/conda.md:22]: > It seems most DeepModeling packages don't support Windows, though...

hahaha that's right

File: faq/conda.md (hunk 2 above)
Comment (njzjz) [reply faq/conda.md]: Oh, I think these posts are pretty old. NVIDIA introduced compatibility in CUDA 11.

Comment (njzjz) [reply faq/conda.md]: See https://docs.nvidia.com/deploy/cuda-compatibility/index.html#deployment-considerations

File: faq/conda.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
+# Conda/pip environments
+
+:::{note}
+The following instructions are for the Linux operator system.
```
Comment (caic99) [inline faq/conda.md]:
```suggestion
The following instructions are for the Linux operating system.
```

**Pull request merged**